# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/sawfish/c23-bool.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Use stdbool.h, bool is a keyword in C23 and cannot be typedef-ed.

--- a/src/sawfish.h	2021-12-24 10:31:19.000000000 +0100
+++ b/src/sawfish.h	2026-09-25 19:34:39.426346607 +0200
@@ -49,7 +49,7 @@
 # define rep_INTERFACE 7		/* rep 0.10 */
 #endif
 
-typedef int bool;
+#include <stdbool.h>
 
 #ifdef rep_HAVE_UNIX
 # define HAVE_UNIX 1
